src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -214,29 +227,44 @@ private bool ValidateRootBuildOptions()
         }
 
         /// <summary>
-        ///     Obtain the root build.
+        ///     Obtain the root builds to start with.
         /// </summary>
-        /// <returns>Root build to start with.</returns>
-        private async Task<Build> GetRootBuildAsync()
+        /// <returns>Root builds to start with, or null if a root build could not be found.</returns>
+        private async Task<IEnumerable<Build>> GetRootBuildsAsync()
         {
-            if (!ValidateRootBuildOptions())
+            if (!ValidateRootBuildsOptions())
             {
                 return null;
             }
 
             IRemote remote = RemoteFactory.GetBarOnlyRemote(_options, Logger);
 
             string repoUri = GetRepoUri();
+            List<int> rootBuildIds = new List<int>();
             if (_options.RootBuildId != 0)
             {
-                Console.WriteLine($"Looking up build by id {_options.RootBuildId}");
-                Build rootBuild = await remote.GetBuildAsync(_options.RootBuildId);
-                if (rootBuild == null)
+                rootBuildIds.Add(_options.RootBuildId);
+            }
+            rootBuildIds.AddRange(_options.RootBuildIds);
+
+            if (rootBuildIds.Any())
```

Comment:
NIT: Perhaps you could rely on `ValidateRootBuildsOptions` have taken care of this at this point.

Reply:
If --id wasn't used, there are still other options.